client/blocks/signup-form/social.jsx

```diff
@@ -76,7 +76,11 @@ class SocialSignupForm extends Component {
 		return (
 			<div className="signup-form__social">
 				{ ! this.props.compact && (
-					<p>{ preventWidows( this.props.translate( 'Or create an account using:' ) ) }</p>
+					<p>
+						{ preventWidows(
+							this.props.translate( 'Or connect your existing profile to get started faster:' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Or connect your existing profile to get started faster.' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/combine-signup-tos-blocks)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).